.github/workflows/build-test-jar.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+
+on: push
```

Comment:
As far as I understand, this workflow hasn't been tested on Github Actions yet. Therefore, I think it should be triggered only manually.

Use [`workflow_dispatch`](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) instead of `push`, like [here](https://github.com/VirtusLab/intellij-pants-plugin-tests/blob/master/.github/workflows/prepare-image.yml#L3). 

After this small change, this PR should be merged regardless of any other faults it might have. This will allow you to trigger the workflow and test it out. It will be safe to merge, because it doesn't seem to affect the existing workflows.

Reply:
It was tested, but the github actions have been spawned in the fork repository